### PR TITLE
[T3A5][F12-A2] Yee Jian Feng, Eric

### PR DESCRIPTION
**Pull request by yeejfe**
extract out the responsibility of formatting text for display from TextUI class

For Address class encapsulation, refer to T2A3

### CODE REVIEW

File: src/seedu/addressbook/ui/Formatter.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+package seedu.addressbook.ui;
+
+import java.io.PrintStream;
+
```
Comment (yeejfe) [inline src/seedu/addressbook/ui/Formatter.java]: Comment

Comment (brandonyeoxg) [reply src/seedu/addressbook/ui/Formatter.java]: ??

File: src/seedu/addressbook/ui/Formatter.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+public class Formatter {
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/ui/Formatter.java]: Do state what this class can do in your header comments.

File: src/seedu/addressbook/ui/Formatter.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    private final PrintStream out;
+
+    
+    public Formatter(PrintStream out) {
+        this.out = out;
+    }
+
+    /** Shows message(s) to the user */
+    public void showToUserNoNewLine(String message) {
+        out.println(LINE_PREFIX + message);
```
Comment (brandonyeoxg) [inline src/seedu/addressbook/ui/Formatter.java]: Is this really SRP?
Recall from the tutorial.

**Review by brandonyeoxg — COMMENTED** [review]
@yeejfe Some comments added. Please close the PR after reading comments.

**Pull request closed without merging**